packages/synapse-react-client/src/components/widgets/query-filter/FacetChip.tsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react'
-import { FacetColumnResult } from '@sage-bionetworks/synapse-types'
 import IconSvg from '../../IconSvg/IconSvg'
-import { useQueryVisualizationContext } from '../../QueryVisualizationWrapper'
 
-export type FacetChipProps = {
-  facet: FacetColumnResult
+export type FacetChipProps = React.PropsWithChildren<{
   isChecked: boolean
   onClick: () => void
-}
+}>
 
-export const FacetChip: React.FC<FacetChipProps> = ({
-  facet,
-  isChecked,
-  onClick,
-}) => {
-  const { getColumnDisplayName } = useQueryVisualizationContext()
+export function FacetChip(props: FacetChipProps) {
```

Comment:
Changed no longer rely on the query context. Instead of getting a label from context, it just renders whatever you pass as a child.